### PR TITLE
Add docs for AWS Glue support in Hudi

### PR DESCRIPTION
**Pull request by alberttwong**
adding documentation that AWS Glue support in Hudi.

## Release notes

(X) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/connector/hudi.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ To use the Hudi connector, you need:
 
 - Hudi version 0.12.3 or higher.
 - Network access from the Trino coordinator and workers to the Hudi storage.
-- Access to a Hive metastore service (HMS).
-- Network access from the Trino coordinator to the HMS.
+- Access to a Hive metastore service (HMS) or AWS Glue.
+- Network access from the Trino coordinator to the HMS or AWS Glue.
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/hudi.md:16]: We don't run CI for AWS Glue with Hudi connector at this time. I hesitate documenting it in this situation.

File: docs/src/main/sphinx/connector/hudi.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ To use the Hudi connector, you need:
 
 - Hudi version 0.12.3 or higher.
 - Network access from the Trino coordinator and workers to the Hudi storage.
-- Access to a Hive metastore service (HMS).
-- Network access from the Trino coordinator to the HMS.
+- Access to a Hive metastore service (HMS) or AWS Glue.
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/hudi.md:15]: > Update hudi.md 

The commit title is too generic. You can remove the commit body and mentioned the change in a title instead.

File: docs/src/main/sphinx/connector/hudi.md (hunk 1 above)
Comment (alberttwong) [reply docs/src/main/sphinx/connector/hudi.md:16]: I've tested it personally that it works.   I don't know if that helps or not.

File: docs/src/main/sphinx/connector/hudi.md (hunk 2 above)
Comment (alberttwong) [reply docs/src/main/sphinx/connector/hudi.md:15]: sounds good.

**Comment by mosabua** [comment]
I rebased and updated the PR now .. wdyt @ebyhr @yihua .. should we merge this even though we don't have tests for it?

**Comment by ebyhr** [comment]
@mosabua Glue isn't tested as I mentioned in https://github.com/trinodb/trino/pull/23093#discussion_r1726349188

**Comment by mosabua** [comment]
Sounds good .. @alberttwong and @yihua .. can you maybe add some testing with Glue in this PR or a separate PR? In the meantime I will close this PR.

**Pull request closed without merging**